src/io/flutter/inspector/FlutterAllMemoryData.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.flutter.inspector;
+
+import com.android.tools.adtui.model.*;
+import com.android.tools.profilers.StudioProfilers;
+import com.android.tools.profilers.memory.MemoryDataSeries;
+import com.intellij.openapi.Disposable;
+import com.intellij.openapi.util.Disposer;
+import io.flutter.perf.HeapMonitor;
+import io.flutter.run.daemon.FlutterApp;
+import org.dartlang.vm.service.element.IsolateRef;
+import org.dartlang.vm.service.element.VM;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
+
+public class FlutterAllMemoryData {
```

Comment:
it might already be formatted but if not, go ahead and run the intellij code formatter on all your new code.